helm/sematic-server/values.yaml

```diff
@@ -22,6 +18,10 @@ database:
   url: postgres://postgresql:5432/postgres # REPLACE ME
 
 deployment:
+  socket_io:
+    dedicated: false
+    worker_count: 1
```

Comment:
This imbrication is kinda misleading. It would suggest you're configuring these many workers for the socketio server, but instead you're activating the socketio server deployment, and instead configuring these many workers for the other server.

How about renaming it to "api_worker_count"? (Not a perfect solution.)

Reply:
I think it should just be de-dented to sit outside the `socket_io` configuration section. Perhaps that is what you are suggesting too, Tudor; I just am unfamiliar with "imbrication" 😆 (though here it seems it's a synonym for "indentation?").

Reply:
👍